src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -439,9 +478,10 @@ def notify_sentry_app(event: Event | GroupEvent, futures):
                 "sentry_app_id": f.kwargs["sentry_app"].id,
                 "settings": settings,
             }
-
         send_alert_event.delay(
-            event=event,
+            instance_id=event.event_id,
+            group_id=event.group_id,
+            occurrence_id=event.occurrence_id if hasattr(event, "occurrence_id") else None,
```

Comment:
This is dangerous. We could end up sending these new signature tasks to workers that haven't been updated yet. I would recommend using an option + `sentry.features.in_random_rollout` so that you can enable use the new signature after the deploy has reached all workers.  #79042 is a recent example of how I did this for a different task.